schema.sql

```diff
@@ -0,0 +1,11 @@
+DROP DATABASE IF EXCISTS reviews
+
+CREATE DATABASE reviews
+
+USE reviews
+
+CREATE TABLE Messages (
```

Comment:
```suggestion
CREATE TABLE messages (
```
Convention is usually to use all lower case for MySQL, up to you if you want to have it capitalized though.

Reply:
Better to stick with what is conventional, i'll go ahead and change it.